### ISSUE TITLE
Use `GenericMvTDist` rather than `MvTDist` when calling getindex on KeyedGenericMvTDist

### ISSUE THREAD

**Issue by glennmoy** [invenia/KeyedDistributions.jl#19]
Sorry, I know it's late already, but I think you need to use `GenericMvTDist` rather than `MvTDist` for various PDMat types.

_Originally posted by @wytbella in https://github.com/invenia/KeyedDistributions.jl/pull/18#discussion_r664778941_

**Comment by mjp98** [comment]
Is this still an issue - the code [here](https://github.com/invenia/KeyedDistributions.jl/blob/main/src/KeyedDistributions.jl#L122) appears to call `GenericMvTDist`? 

```Julia
function Base.getindex(d::KeyedGenericMvTDist, i::Vector)::KeyedGenericMvTDist
    return KeyedDistribution(GenericMvTDist(d.d.df, d.d.μ[i], submat(d.d.Σ, i)), axiskeys(d)[1][i])
end
```

**Comment by glennmoy** [comment]
Yeah it was solved in #30 but didn't close this issue
